docs/tutorials/kubernetes/outside-cluster.md

```diff
@@ -0,0 +1,9 @@
+# Non In-Cluster Execution
```

Comment:
I prefer to call this differently, like External Access, what do you think?

Reply:
Me too, I agree with you